CHANGES.md

```diff
@@ -1,6 +1,7 @@
 Unreleased
 ---
 * [BREAKING] Make `have_enqueued_sidekiq_job()` match jobs with any arguments (same as `enqueue_sidekiq_job()` or `have_enqueued_sidekiq_job(any_args)`) ([@3v0k4](https://github.com/3v0k4) #215)
+* Add support for expected number of jobs to both `enqueue_sidekiq_job` and `have_enqueued_sidekiq_job` ([@3v0k4](https://github.com/3v0k4) #219)
```

Comment:
🎨 mind just listing the PR number?

That's been the style for the change log for the last several years on this repo. You'll still be listed as the PR author and as the contributor on the release notes.